viewing_party/party.py

```diff
@@ -65,17 +65,72 @@ def watch_movie(user_data, title):
             list_of_watched_movie_dict.append(movie_dict)
             # exit loop early if found
             break
+
     return user_data
 
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
 
+#wave 2 
 
-# -----------------------------------------
-# ------------- WAVE 3 --------------------
-# -----------------------------------------
+def get_watched_avg_rating(user_data):
+    """
+    the value of user_data will be a dictionary with a "watched" list of movie dictionaries
+    This represents that the user has a list of watched movies
+    Calculate the average rating of all movies in the watched list
+    The average rating of an empty watched list is 0.0
+    return the average rating
+    user_data = {"watched": [...list of movie_dict watched...]}
+    """
+
+    watched_list = user_data["watched"]
+    total_rating = 0
+    
+    if watched_list == []:

```

Comment:
Hi Ashley,
Sorry about repeating this issue
The more Pythonic way would be:
if not lod_watched: versus what I have here